### DISCUSSION TITLE
Visualizing the graph nodes and edges

### DISCUSSION THREAD

**Discussion by shivarajmulimani** [cozodb/cozo#279] [Q&A]
Why there is no visualisation available, is there any package or visualise graph data in Cozodb?

I would be happy to contribute this piece of code.

**Comment by ConstantinVasilev** [comment]
I would be interested to work on visualisations with cytoscape but my tech stack is mainly Python and I don't even know if I can easily interact with cozo from a notebook for example.

Here is an interesting open-source graph visualisation studio. If the 2 could be linked that would also be great
https://github.com/whyhow-ai/knowledge-graph-studio

**Reply by elacosse** [reply]
I second this.

**Comment by adiibanez** [comment]
What about sigma.js
There is a neo4j integration https://github.com/sim51/neosig
